packages/api-reference/src/features/Operation/components/ParameterListItem.vue

```diff
@@ -74,12 +74,12 @@ const shouldShowParameter = computed(() => {
         v-if="shouldCollapse"
         class="parameter-item-trigger"
         :class="{ 'parameter-item-trigger-open': open }">
-        <ScalarIconCaretRight
-          weight="bold"
-          class="parameter-item-icon size-3 transition-transform duration-100"
-          :class="{ 'rotate-90': open }" />
         <span class="parameter-item-name">
-          {{ parameter.name }}
+          <ScalarIconCaretRight
```

Comment:
This moves the caret to be centered to the status code instead of the whole block

Before:
<img width="3018" height="1968" alt="Image" src="https://github.com/user-attachments/assets/bf2e9579-2911-476f-938f-29ae607521f4" />